### PR TITLE
Slight improvement to clarity of Nodes.md

### PR DESCRIPTION
**Pull request by alejandroEsc**
changes to node.md for clarity since sections and subsections visually are NOT that different in sizes and single line comment was not clear enough and looked incomplete, specially at first read. Added .idea/ directory in gitignore.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1925)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ _site/**
 .sass-cache/**
 CNAME
 .travis.yml
+.idea/
```
Comment (kelseyhightower) [inline .gitignore]: Why is this hear? I think we should remove this from the commit.

**Comment by kelseyhightower** [comment]
One minor nitpick, but otherwise looks good to me.

File: .gitignore (hunk 1 above)
Comment (alejandroEsc) [inline .gitignore]: I like to use idea to work on the markdown text (its actually pretty nice). This makes sure no one accidentally commits idea related files into the repo. Fairly harmless addition but i can remove it if you wish.

**Comment by alejandroEsc** [comment]
@kelseyhightower thanks for reviewing! I replied to your comment, let me know if you are ok with that or rather have me remove the line still.

File: .gitignore (hunk 1 above)
Comment (kelseyhightower) [inline .gitignore]: Please push that change in another commit. Also please squash this change into a single commit. Thanks.

Comment (alejandroEsc) [inline .gitignore]: sounds good, thank you!

**Comment by alejandroEsc** [comment]
@kelseyhightower done. once again, thank you!

**Comment by alejandroEsc** [comment]
@kelseyhightower changes you request have been completed. Please have another look when you are ready. Thank you!

**Comment by alejandroEsc** [comment]
*[.gitignore, line 8 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/1925#-KYnyOIBSpaDGR9Sluo9-r1-8:-KYzPH0pe90ikH8QcmwI:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes.github.io/blob/f4fe31314ccb534c7116bb03112e31cbeffa42f9/.gitignore#L8)):*
<details><summary><i>Previously, alejandroEsc (Alejandro Escobar) wrote…</i></summary><blockquote>

sounds good, thank you!
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/1925)*

File: docs/admin/node.md [hunk 2]
```diff
@@ -20,7 +20,15 @@ architecture design doc for more details.
 
 ## Node Status
 
-A node's status is comprised of the following information.
+A node's status is comprised of the following information
```
Comment (devin-donnelly) [inline docs/admin/node.md]: "A node's status contains the following information:"

File: docs/admin/node.md [hunk 3]
```diff
@@ -20,7 +20,15 @@ architecture design doc for more details.
...
+
+* [Addresses](#Addresses)
+* ~~[Phase](#Phase)~~ **deprecated**
+* [Condition](#Condition)
+* [Capacity](#Capacity)
+* [Info](#Info)
+
+with details described in the subsections below.
```
Comment (devin-donnelly) [inline docs/admin/node.md]: "Each section is described in detail below."

**Comment by alejandroEsc** [comment]
@devin-donnelly thank you for the great recommendations. The changes are complete. Are you able to merge these in?

**Pull request merged**